test/src/e2e_vm_tests/mod.rs

```diff
@@ -343,6 +343,11 @@ pub fn run(filter_regex: Option<regex::Regex>) {
             "should_pass/test_contracts/test_fuel_coin_contract",
             "should_pass/require_contract_deployment/token_ops_test",
         ),
+        /* Requires IR - disable until the IR pipeline is enabled by default
```

Comment:
Add a TODO here so it's easily searchable at least, maybe even file an issue.

Reply:
Done.